admin/sql/create_extensions.sql

```diff
@@ -1 +1,3 @@
 CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+CREATE EXTENSION IF NOT EXISTS "pg_trgm";
+ALTER DATABASE listenbrainz SET pg_trgm.word_similarity_threshold = 0.1;
```

Comment:
The default threshold is 0.3 which eliminates a lot of possible matches.

Reply:
I believe that -- our "documents" are quite short in comparison with what others might put into a document.